exec/jdbc-all/pom.xml

```diff
@@ -562,7 +562,7 @@
                   This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
 
                   </message>
-                  <maxsize>46600000</maxsize>
+                  <maxsize>50000000</maxsize>
```

Comment:
Is there any reason for updating this limit? The functionality is only for HTTP plugin, so it shouldn't affect jdbc driver size.

Reply:
I'm not sure why this happened.  I did bump the version of `okhttp3` to 4.9.1 or whatever the latest version is, but there are no new dependencies.

Reply:
Before your change, okhttp3 was present only for test scope (and used only in some unit tests), and therefore it wasn't included in the JDBC driver. But you have changed its scope from the test to compile, so it is added to the driver. But the question here: is it required in JDBC?

Reply:
@vvysotskyi Thanks for looking at this.  It actually is used in one of the files for Drill's internal rest API.  (See below). Does that need to be included in the JDBC driver?

https://github.com/apache/drill/blob/d6c54936129f8e2a8eda83be7b1ab69998e893f8/exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/StorageResources.java#L206-L12

Reply:
I don't think it should be included, so please exclude it from the driver.

Reply:
@vvysotskyi Fixed!

Reply:
@vvysotskyi Unfortunately I tried excluding it from the JDBC and it does need to be there.